package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "sequelize": "^3.30.2",
     "sequelize-cli": "^2.5.1",
+    "split.js": "1.3.5",
```

Comment:
@jdaguil  Per our recent conversation, when I remove the [CSS for the splitter's gutter icons](https://github.com/nexB/aboutcode-manager/blob/21-resize-views/assets/css/main.css#L305-L313), on Windows 10 I no longer see a visual cue (the `ew-resize` cursor) when I pass the cursor over the vertical gutter.  Although I can still drag the gutter east/west, I think the lack of both the gutter icon and the mouseover cue will leave many users unaware that the views can be resized.

I'm going to test instead keeping and modifying those two CSS classes to something innocuous to see if the effect will be to remove the gutter icon you want removed while preserving the `ew-resize` cursor as a cue to the user.